### PR TITLE
css-brochure-form - Sean Kim

### PR DESCRIPTION
**Pull request by seankim248**
Screen Shot 2021-12-01 at 8 27 44 PM

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <input type="text" id="name" placeholder="John Smith" />
+      </div>
+      <div>
+        <label for="location">WHERE ARE YOU HIRING?</label>
+        <input type="text" id="location" placeholder="City, Country" />
+      </div>
+      <div>
+        <label for="hire"
+          >HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label
+        >
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:30]: Not sure what happened here, but your indentation got a bit wacky 😬

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
One minor cleanup note!

File: css-brochure-form/index.html (hunk 1 above)
Comment (seankim248) [reply css-brochure-form/index.html:30]: Yeah, that's been happening after I save a file I think. Not too sure what's causing it but I'll try and find out.

**Review by codingcodymiller — APPROVED** [review]
Excellent work!!

**Pull request merged**